Pipfile

```diff
@@ -14,7 +14,9 @@ molecule = "*"
 python_version = "2.7"
 
 [scripts]
-test ="molecule test"
+clean = "molecule destroy"
```

Comment:
Hi @rshad ,

Think we should move to another issue to keep better track of changes related to test

Thanks!